contrib/ansible/vagrant/openstack_config.yml

```diff
@@ -0,0 +1,12 @@
+os_username: eparis
```

Comment:
I do use crappy passwords, but THAT one isn't one of my crappy passwords   :)

I'll move to the .example version and make the .yml in .gitignore.